doc/nrf/libraries/others/profiler.rst

```diff
@@ -13,130 +13,196 @@ The output is provided using RTT and can be visualized in a custom Python backen
 
 See the :ref:`profiler_sample` sample for an example of how to use the Profiler.
 
+.. _profiler_configuration:
+
 Configuration
 *************
 
-Apart from standard configuration parameters, there is one required setting:
-
-:kconfig:`CONFIG_PROFILER`
-  Set this option to add the Profiler source code to the application.
-  If you use the Event Manager, you can also set this option by selecting :kconfig:`CONFIG_EVENT_MANAGER_PROFILER`.
-  Please read :ref:`event_manager` to see how to declare events for profiling perpouse.
+To use the Profiler for custom events, enable the :kconfig:`CONFIG_PROFILER` Kconfig option.
```

Comment:
If we describe how to configure profiler in next steps, this is redundant.

Reply:
Or it should be moved to Configuring profiling for custom events part

Reply:
Reworked, thanks.